README.md

```diff
@@ -151,47 +142,19 @@ Tenjin.getAnalyticsInstallationId()
 ```
 Returns: callback -> `string`
 
-### Send AdMob impression (ILRD)
-```javascript
-Tenjin.eventAdImpressionAdMob(json)
-```
-Parameters:
-- `json`: JSON
+### Impression Level Revenue Data Integration (ILRD)
+Tenjin supports the ability to integrate with the Impression Level Ad Revenue (ILRD) feature from,
 
-### Send AppLovin impression (ILRD)
-```javascript
-Tenjin.eventAdImpressionAppLovin(json)
-```
-Parameters:
-- `json`: JSON
+- AppLovin
+- Unity LevelPlay
+- AdMob
+- TopOn
+- Clever Ads Solutions (CAS)
+- TradPlus
+This feature allows you to receive events which correspond to your ad revenue is affected by each advertisement show to a user. To enable this feature, follow the below instructions.
```

Comment:
@makototaguchi1110 Should we maybe remove this part? "To enable this feature, follow the below instructions."

Reply:
@giraldogdiego thanks, updated